wsmux/post-test-data

```diff
@@ -0,0 +1,2 @@
+This string is supposed to be sent over 
```

Comment:
If this is a test file you really should put it into a `testdata/` subfolder... as is golang convention...

But really, there is no need to have a file for such a small string... you can just embed it,.. if you want a stream you can use `bytes.NewBuffer`...